varcode/effect_ordering.py

```diff
@@ -52,10 +52,18 @@ def top_priority_transcript_effect(effects):
         elif priority == best_priority:
             best_effects.append(effect)
 
-    def effect_cds_len(effect):
-        """
-        Returns length of coding sequence of transcript associated with effect
-        """
-        return len(effect.transcript.coding_sequence)
-
-    return max(best_effects, key=effect_cds_len)
\ No newline at end of file
+    if any(effect.transcript.complete for effect in best_effects):
```

Comment:
I guess the question is: why not leave the non-coding transcript stuff in there (when transcripts do have complete coding sequence annotations), and just have it lower in the sort order vs. filtering it out? Not saying you should. Just making sure I understand the choices.